<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>singU</class>
 <widget class="QMainWindow" name="singU">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(146, 230, 255);color:rgb(255, 0, 0) ;
font: italic 14pt &quot;Segoe UI&quot;;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="formLayoutWidget">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>110</y>
      <width>281</width>
      <height>281</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="styleSheet">
        <string notr="true">color:rgb(255, 0, 0) ;
font: italic 14pt &quot;Segoe UI&quot;;</string>
       </property>
       <property name="text">
        <string>UserName:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="userLine">
       <property name="styleSheet">
        <string notr="true">color:rgb(255, 0, 0) ;
font: italic 14pt &quot;Segoe UI&quot;;</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="styleSheet">
        <string notr="true">color:rgb(255, 0, 0) ;
font: italic 14pt &quot;Segoe UI&quot;;</string>
       </property>
       <property name="text">
        <string>Password:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="passLine">
       <property name="styleSheet">
        <string notr="true">color:rgb(255, 0, 0) ;
font: italic 14pt &quot;Segoe UI&quot;;</string>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="styleSheet">
        <string notr="true">color:rgb(255, 0, 0) ;
font: italic 14pt &quot;Segoe UI&quot;;</string>
       </property>
       <property name="text">
        <string>firstName:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="firstNLine">
       <property name="styleSheet">
        <string notr="true">color:rgb(255, 0, 0) ;
font: italic 14pt &quot;Segoe UI&quot;;</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>lastName:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="lastNLine">
       <property name="styleSheet">
        <string notr="true">color:rgb(255, 0, 0) ;
font: italic 14pt &quot;Segoe UI&quot;;</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_4">
       <property name="styleSheet">
        <string notr="true">color:rgb(255, 0, 0) ;
font: italic 14pt &quot;Segoe UI&quot;;</string>
       </property>
       <property name="text">
        <string>gmail:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="gmailLine"/>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>security Key</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="SkeyLine"/>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>400</y>
      <width>161</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(255, 0, 0) ;
font: italic 14pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="text">
     <string>sign Up</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>420</y>
      <width>131</width>
      <height>29</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(255, 0, 0) ;
font: italic 14pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="text">
     <string>Back to Login</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>36</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
